repeat.py

```diff
@@ -0,0 +1,4 @@
+repeat_str = ('name')
```

Comment:
What is this repeat_str = ('name')? Fix it

Reply:
Fixed